pkg/actors/internal/placement_service.go

```diff
@@ -22,7 +22,7 @@ import (
 type PlacementService interface {
 	io.Closer
 
-	Start(ctx context.Context) error
+	Start(context.Context)
```

Comment:
Please leave the error. This was added recently and it helps with future expansion

```suggestion
	Start(ctx context.Context) error
```